ts/Extensions/Data.ts

```diff
@@ -394,47 +400,39 @@ class Data {
      *
      * @function Highcharts.Data#getColumnDistribution
      */
-    public getColumnDistribution(): void {
+    public getColumnDistribution(): asserts this is this & {
+        valueCount: DataValueCountObject
```

Comment:
How about initializing the prop with 0 instead?

Reply:
Sadly, this won't help as far as I can see it because `valueCount` is of type:
```ts
interface DataValueCountObject {
    global: number;
    globalPointArrayMap: Array<string>;
    individual: Array<number>;
    seriesBuilders: Array<SeriesBuilder>;
    xColumns: Array<number>;
}
```
and later running function (`complete`) needs it.

Reply:
Okay, what a mess.